internal/tfengine/tfengine.go

```diff
@@ -336,3 +338,34 @@ func findUnmanaged(generatedDir, outputDir string, deleteFiles bool) error {
 
 	return nil
 }
+
+// trimDescriptions remove tabs and extra space from schema descriptions
+// so they can be used during rendering without unexpected gaps between lines
+func trimDescriptions(schema map[string]interface{}) {
+	val := reflect.ValueOf(schema)
+
+	for _, key := range val.MapKeys() {
+		v := val.MapIndex(key)
+
+		switch t := v.Interface().(type) {
+		case map[string]interface{}:
+			trimDescriptions(t)
+		case string:
+			if key.String() == "description" {
+				description := v.Interface().(string)
+				trimmedDescription := strings.TrimSpace(leftTrim(description))
+				schema["description"] = trimmedDescription
+			}
+		}
+	}
+}
+
+// leftTrim trims leading space from every line.
+func leftTrim(s string) string {
```

Comment:
Function looks fine but what is the purpose of removing leading spaces from every line?

Reply:
Oh I realized there's an issue linked. I'll resolve this while I read that.